core/templates/dev/head/pages/story_editor/StoryEditor.ts

```diff
@@ -77,17 +77,23 @@ require('domain/editor/undo_redo/UndoRedoService.ts');
 require('domain/utilities/UrlInterpolationService.ts');
 require('pages/story_editor/StoryEditorStateService.ts');
 require('services/contextual/UrlService.ts');
+require('services/PageTitleService.ts');
 
 oppia.constant('NODE_ID_PREFIX', 'node_');
 
+oppia.constant('EVENT_STORY_INITIALIZED', 'storyInitialized');
+oppia.constant('EVENT_STORY_REINITIALIZED', 'storyReinitialized');
+
```

Comment:
Don't add these constants again, they are defined in `StoryEditorStateService.ts`

Reply:
done